src/lib/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,90 @@
+import styled from "@emotion/styled";
+
+type AvatarSizes = "extra large" | "large" | "medium" | "small" | "tiny";
+type AvatarStatus = "online" | "offline";
+
+export interface AvatarProps {
+  name: string;
+  size: AvatarSizes;
+  image?: string;
+  status?: AvatarStatus;
+}
+
+const diameter: { [key in AvatarSizes]: string } = {
+  "extra large": "80px",
+  large: "64px",
+  medium: "48px",
+  small: "32px",
+  tiny: "16px",
+};
+
+const fontSize: { [key in AvatarSizes]: string } = {
+  "extra large": "1.75rem",
+  large: "1.4rem",
+  medium: "1.05rem",
+  small: "0.7rem",
+  tiny: "0.35rem",
+};
+
+const fontWeight: { [key in AvatarSizes]: string } = {
```

Comment:
I'm not sure that we really need the `fontWeight` object. Maybe we can reuse values from a `theme` or just set it in styles.

Reply:
you are absolutely right, i would just set it as 800 for every size as i think this weight looks best